mathesar/tests/rpc/test_columns.py

```diff
@@ -0,0 +1,131 @@
+from contextlib import contextmanager
+
+from mathesar.rpc import columns
+from mathesar.models.users import User
+
+
+def test_columns_list(rf, monkeypatch):
```

Comment:
Can you please add a docstring for the fixtures used here?

Reply:
Good catch. I'll add a docstring at the top of the file. I think that's going to be better in general, since fixtures are defined in such a way that they're file-wide and not restricted to any given test function.